docs/gatsby-config.js

```diff
@@ -8,21 +8,28 @@ module.exports = {
       options: {
         ...themeOptions,
         root: __dirname,
-        subtitle: 'Rover CLI',
+        subtitle: 'Rover CLI (Preview)',
```

Comment:
Should we drop the `CLI` in the name or do you think that adds clarity @StephenBarlow?

Reply:
I think we should keep "CLI" for the time being, until Rover has better name recognition via full release. Will be especially helpful in the docset switcher